edflow/edflow

```diff
@@ -120,6 +126,12 @@ def main(opt):
     # Evaluation
     opt.eval = opt.eval or list()
     for eval_idx, eval_config in enumerate(opt.eval):
+        if opt.checkpoint is not None:
+            checkpoint = opt.checkpoint
+        # elif opt.project is not None:
+        #     checkpoint = get_latest_checkpoint(P.checkpoints)
+        else:
+            checkpoint = None
```

Comment:
Maybe:

    else:
        checkpoint = None
        if opt.project is not None:
            checkpoint = get_latest_checkpoint(P.checkpoints)

This is useful for iterators other than the tf evaluator to then simply get the latest checkpoint path.

Reply:
should be fixed